models/amat2_psc_exp.h

```diff
@@ -350,6 +350,16 @@ class amat2_psc_exp : public Archiving_Node
   {
     return S_.V_th_v_;
   }
+  double
+  get_I_syn_ex_() const
+  {
+    return S_.I_syn_ex_;
+  }
+  double
```

Comment:
Insert and empty line before the function definition. Also, I would suggest to add an `inline`.